src/hooks/useSelectedTransactionsActions.ts

```diff
@@ -0,0 +1,118 @@
+import {useMemo} from 'react';
+import * as Expensicons from '@components/Icon/Expensicons';
+import {useMoneyRequestReportContext} from '@components/MoneyRequestReportView/MoneyRequestReportContext';
+import {deleteMoneyRequest, unholdRequest} from '@libs/actions/IOU';
+import {exportReportToCSV} from '@libs/actions/Report';
+import Navigation from '@libs/Navigation/Navigation';
+import {getIOUActionForTransactionID, getOriginalMessage, isDeletedAction, isMoneyRequestAction} from '@libs/ReportActionsUtils';
+import {canDeleteCardTransactionByLiabilityType, canDeleteTransaction} from '@libs/ReportUtils';
+import {getTransaction, isOnHold} from '@libs/TransactionUtils';
+import CONST from '@src/CONST';
+import ROUTES from '@src/ROUTES';
+import type {OriginalMessageIOU, Report, ReportAction, Session} from '@src/types/onyx';
+import useLocalize from './useLocalize';
+
+// We do not use PRIMARY_REPORT_ACTIONS or SECONDARY_REPORT_ACTIONS because they weren't meant to be used in this situation. `value` property of returned options is later ingored.
```

Comment:
```suggestion
// We do not use PRIMARY_REPORT_ACTIONS or SECONDARY_REPORT_ACTIONS because they weren't meant to be used in this situation. `value` property of returned options is later ignored.
```